impl/maven-cli/src/main/java/org/apache/maven/cling/invoker/CommonsCliOptions.java

```diff
@@ -472,24 +472,28 @@ public void displayHelp(String command, Consumer<String> pw) {
 
             pw.accept("");
 
-            StringWriter sw = new StringWriter();
-            PrintWriter pw2 = new PrintWriter(sw);
-            formatter.printHelp(
-                    pw2,
-                    width,
-                    commandLineSyntax(command),
-                    System.lineSeparator() + "Options:",
-                    options,
-                    HelpFormatter.DEFAULT_LEFT_PAD,
-                    HelpFormatter.DEFAULT_DESC_PAD,
-                    System.lineSeparator(),
-                    false);
-            pw2.flush();
-            for (String s : sw.toString().split(System.lineSeparator())) {
+            for (String s : getStringWriter(command, formatter, width).toString().split(System.lineSeparator())) {
                 pw.accept(s);
```

Comment:
what even happens here? is this some kind of pw password matching? or pw pringWriter idk?